Jenkinsfile.cloud

```diff
@@ -89,10 +89,12 @@ node(NODE) {
         bwrap --bind / / --dev-bind /dev /dev --proc /proc --unshare-pid runuser -u builder -- coreos-virt-install --dest ${image} --create-disk \
                 --console-log-file ${WORKSPACE}/rhcos-qcow2-install.txt \
                 --wait=25 --kickstart cloud.ks --location "${INSTALLER_TREE_URL}"
+        cp ${image} ${WORKSPACE}
         """
     } }
     } finally {
         archiveArtifacts artifacts: "rhcos-qcow2-install.txt", allowEmptyArchive: true
+        archiveArtifacts artifacts: "rhcos.qcow2", allowEmptyArchive: true
```

Comment:
How about something like this in the "Kola rhcos.basic" step:

```
stage ... {
  kola version
  if ! kola -b rhcos ...; then
    cp ${image} ${WORKSPACE}
    exit 1
  fi
} finally {
  ...
  archiveArtifacts artifacts: "rhcos.qcow2", allowEmptyArchive: true
}
```

?

That way we only copy it and archive it if the test failed. If the test passed, then we'll already have access it to it at the usual place.

Reply:
This is smart.  I will explore this.  Thanks!